### PR TITLE
Interstitital Mode - disable teleport and fix avatar outside of interstitial bubble

### PR DESCRIPTION
**Pull request by danteruiz**
This PR disables teleport during the interstitial mode and potential fix of avatar outside interstitial bubble

tickets
https://highfidelity.manuscript.com/f/cases/19292
https://highfidelity.manuscript.com/f/cases/19294

### CODE REVIEW

**Comment by danteruiz** [comment]
Test plan
1.
- you should not be able to teleport while in interstitial mode.

2. 
Steps to Reproduce: Unknown  -below is an estimated guess.
1) Log into an account,
2) Log out, then back into a second player account.
3) Change domain, Move away from spawn by using the warp.
4) Repeat 3 and 2 for as many time you wish.
5) Interstitial bubble should not be displaced.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13821/HighFidelity-Beta-PR14188-451142f.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13821/HighFidelity-Beta-PR14188-451142f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13821/HighFidelity-Beta-Interface-PR14188-451142f.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13958/HighFidelity-Beta-PR14188-451142f.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13958/HighFidelity-Beta-PR14188-451142f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13958/HighFidelity-Beta-Interface-PR14188-451142f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13958/HighFidelity-Beta-PR14188-451142f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13958/HighFidelity-Beta-Interface-PR14188-451142f.exe

File: scripts/system/interstitialPage.js [hunk 1]
```diff
@@ -545,7 +545,11 @@
     MyAvatar.sensorToWorldScaleChanged.connect(scaleInterstitialPage);
     MyAvatar.sessionUUIDChanged.connect(function() {
         var avatarSessionUUID = MyAvatar.sessionUUID;
-        Overlays.editOverlay(loadingSphereID, { parentID: avatarSessionUUID });
+        Overlays.editOverlay(loadingSphereID, {
+            position: Vec3.sum(Vec3.sum(MyAvatar.position, {x: 0.0, y: -1.0, z: 0.0}), Vec3.multiplyQbyV(MyAvatar.orientation, {x: 0, y: 0.95, z: 0})),
```
Comment (wayne-chen) [inline scripts/system/interstitialPage.js]: I would suggest putting the position calculation into a variable.

**Comment by wayne-chen** [comment]
merged into PR #14249.

**Pull request closed without merging**